runsc/cgroup/cgroup_v2.go

```diff
@@ -170,6 +171,38 @@ func (c *cgroupV2) Install(res *specs.LinuxResources) error {
 	return nil
 }
 
+// Update updates the cgroup resources.
+func (c *cgroupV2) Update(res *specs.LinuxResources) error {
```

Comment:
line 142-167 can be replaced with just call to `c.Update(res)`. The code is identical.